Mesh4all/m4a-firmware
# firmware: fixed set ipv6 address conflicts between RPL, radio and border_router modules

### Contribution description
This PR has to the propose of fix the conflicts presents in ipv6 setup in each module reported in the issue #257, radio will be modify to work under the physical layer, rpl and border router could be needs the settings of ipv6 address, these needs to check if a global address already exists, if doesn't exist don't should be able to modify the ipv6 params. 



### Testing procedure

could add The basic modules as border_router and rpl in makefile
```makefile
USEMODULE += border_router
USEMODULE += rpl_protocol
```
When these modules are added you could manage all the functions required from both elements.
add to .c
```c
....
#include<border_router.h>
#include<rpl_protocol>
...
main(){
    ipv6_addr_t ip;
    ipv6_addr_from_str(&ip_addr, "2001:db8:1::1");
    border_router_add_ipv6(_UNICAST, &ip_addr, WIRELESS_INTERFACE);
    rpl_setup(DODAG); 
}
```
*note* : in case that you are running first `rpl_setup` before to the `border_router_add_ipv6` , you could see that the ipv6 address saved it's the ipv6 addres generated by the uniqueid module or property from cpuid. 

New features:
`rpl_setup()` receives now a parameter to set in mode DAG or DODAG.
`Border_router_add_ipv6()` has a new param to set to a WIRELESS_INTERFACE or WIRED_INTERFACE an ipv6 address-



### Issues/PRs references
Fix #257
